### PR TITLE
Story 2 task 3

### PR DESCRIPTION
**Pull request by robin-hodge**
Run src/Hydrators/ProjectHydratorDemo/demo.php to test it if you want

### CODE REVIEW

File: src/Hydrators/ProjectHydrator.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+// uncomment the below when ready to replace the above
+// use ProjectManagementApi\DatabaseConnection; 
+// use ProjectManagementApi\Entities\Project; 
+
+class ProjectHydrator
+{
+    public static function getAllProjects(DatabaseConnection $db)
+    {
+        $query = $db->prepare('SELECT id, `name`, client_id, deadline FROM projects');
```
Comment (chloe-smart) [inline src/Hydrators/ProjectHydrator.php]: why does name have back ticks?

Comment (west-country) [inline src/Hydrators/ProjectHydrator.php]: backticks?

File: src/Hydrators/ProjectHydrator.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        $query->execute();
```
Comment (west-country) [inline src/Hydrators/ProjectHydrator.php]: is prepare/execute being used (without binding)

Comment (west-country) [reply src/Hydrators/ProjectHydrator.php]: technically don't need prepare and execute, just ->query. But no functional difference.

**Review by chloe-smart — APPROVED** [review]
Great work:)

**Pull request merged**